### DISCUSSION TITLE
error in fetching the workflow file

### DISCUSSION THREAD

**Discussion by abdul-at** [community/community#40986] [Actions]
### Select Topic Area

Bug

### Body

We are getting strange random pipeline failures with error **error in fetching the workflow file**. Work most of the time but failing randomly.

**Invalid workflow file
error parsing called workflow "ORG/org-reusable-workflows/.github/workflows/ci-go.yml@master": error in fetching the workflow file**

**Comment by ossanna16** [comment]
Hi there @abdul-at and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
